src/tv2-common/__tests__/jinglePartProperties.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { GetJinglePartPropertiesFromTableValue } from 'tv2-common'
+
+describe('GetJinglePartPropertiesFromTableValue', () => {
+	it('Calculates values correctly', () => {
```

Comment:
By saying you are testing `values` you indicate you are testing more than one thing which violates SRP and the general principle of unit test. (Also if this test fails, we don't know what value wasn't correct and we would now have to debug. Hopefully the assert message would tell us)

In your `toEqual` you are asserting for four different attributes. If we where to follow unit testing to the letter you should have one test testing that the `expectedDuration` has the correct output. One test for `autoNextOverlap` and etc. (You should actually have multiple test for each testing different scenarios.)

It is however a tricky balance of how many/few test to write

Reply:
"The key is that you have only one action, and then you inspect the results of that action using asserts."
Jest will show you precisely what values were expected and what was the actual result. That is enough information for me.